symbiflow-vtr-optimized/condarc

```diff
@@ -0,0 +1,2 @@
+channels:
+  - conda-forge
```

Comment:
What are you pulling in from `conda-forge`?

Reply:
It's for `python-constraint`: https://travis-ci.com/github/SymbiFlow/conda-packages/jobs/441871052